atlp-rwanda/atlp-pulse-fn
# #DP-111 fix: Update profile picture

### What does this PR do?
This pull request allow the user to update their profile picture

### Description of the tasks to be completed
- [ ] A user should be able to change their avatar
 
### How should this be manually tested?
Use the heroku-review app linked to this pull request or click the link below:

https://devpulse-review-app-99.herokuapp.com/

To test on your local machine:

1. Make sure you clone the project (Backend and Frontend)
2. Checkout the respective branches for both the backend and frontend
3. Follow the steps below for the frontend

```js
- Use the review app.
- use `andela` as an organization to log in.
- login a user.
- change your profile image and cover image.

```
### Pre-requisites:
- use the review apps.

### Relevant trello story:
[#DP-111](https://trello.com/c/AYiItUdt/111-profile-picture-update)
